app/Services/GithubWebhookService.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use App\DTO\OrganizationDTO;
+use Illuminate\Http\Request;
+
+class GithubWebhookService
+{
+    protected $organizationService;
+
+    public function __construct()
+    {
+        $this->organizationService = new OrganizationService();
+    }
+
+    public function handle(Request $request): void
```

Comment:
```suggestion
    public function handleRequest(Request $request): void
```
maybe that method name?

Reply:
changed name

Reply:
I do not see